src/site/markdown/server/manage-webadmin.md

```diff
@@ -493,7 +493,9 @@ firewall rules.
 
 `trustMessageProjection` query parameter can be set to `true`. This will result in a better performance running the
 task at the cost of safety in the face of message denormalization inconsistencies. Defaults to false, which generates 
-additional checks.
+additional checks. You can read 
```

Comment:
Am I the only to _not understand_ what all that means? 

We provide a task to fix inconsistencies in some specific message projections (counters) and we have a parameter `trustMessageProjection`. The comment to describe that comment doesn't describe which specific projection it will allows the task to trust to rebuild the counters projection. We have plenty of message projections AFAIK. And the ADR linked doesn't describe that parameter neither.

I'm lost.

Reply:
Then the ADR is not clear enough? Though it is supposed to describe in details the issue and had been validated by the team.

Let me have another try at it.

Reply:
```
`trustMessageProjection` query parameter can be set to `true`. Cpntent of `messageIdTable` (listing messages by their 
mailbox context) table will be trusted and not compared against content of `imapUidTable` table (listing messages by their
messageId mailbox independent identifier). This will result in a better performance running the
task at the cost of safety in the face of message denormalization inconsistencies. 

Defaults to false, which generates 
additional checks. You can read 
[this ADR](https://github.com/apache/james-project/blob/master/src/adr/0022-cassandra-message-inconsistency.md) to 
better understand the message projection and how it can become inconsistent. 
```

That's my latest go.

If s it more intelligible?

If not please precise clearly which part you don't understand.

Reply:
Great, thank you for that